src/SignaturePad.InkPresenter.Shared/InkPresenter.cs

```diff
@@ -103,15 +119,18 @@ public void Clear ()
 		{
 			paths.Clear ();
 			currentPath = null;
-
+#if !NET471
 			this.Invalidate ();
+#endif
```

Comment:
Is it safe to just remove this invalidation? - and the ones below?

Reply:
I have added it to extension method, so #if is redundant